packages/radix-vue/src/Popper/PopperArrow.vue

```diff
@@ -67,6 +69,8 @@ const baseSide = computed(() =>
       :style="{
         display: 'block',
       }"
+      :as="as"
+      :as-child="props.asChild"
```

Comment:
Nice.. this is the missing piece here! thanks for that @D-Lite ! Perhaps let replace `props.asChild` with `asChild`?

Reply:
Great! made the fix and updated the pr